synfig-studio/src/gui/resourcehelper.cpp

```diff
@@ -57,13 +57,17 @@ synfig::String studio::ResourceHelper::get_image_path()
 	char* synfig_root=getenv("SYNFIG_ROOT");
 	if(synfig_root) {
 		imagepath=synfig_root;
-		// Only class About didn't use the synfigstudio directory when using
+		//  Only class About didn't use the synfigstudio directory when using
 		//  SYNFIG_ROOT env variable. However, if it weren't set, it would use
 		//  IMAGE_DIR builtin variable that includes "synfigstudio". It means
 		//  that that About icon is in both folders. Therefore, it is safe to
 		//  choose this path.
 		imagepath+="/share/pixmaps/synfigstudio";
 	}
+	else {
+	    imagepath=CMAKE_BUILD_DIR;
```

Comment:
This will not work in the situation when you copy binary to another place. 
I suggest to use `App::get_base_path()` ot get base path :)

Reply:
That part is already refactored.

Reply:
I see :) But just try to copy `output` folder to another place, and you will see what it stops working.
So i suggest to fix it another way.